diff --git a/node_modules/ejs/lib/ejs.js b/node_modules/ejs/lib/ejs.js
index aa6322e..756a41b 100755
--- a/node_modules/ejs/lib/ejs.js
+++ b/node_modules/ejs/lib/ejs.js
@@ -104,7 +104,7 @@ exports.localsName = _DEFAULT_LOCALS_NAME;
  * @public
  */
 
-exports.promiseImpl = (new Function('return this;'))().Promise;
+exports.promiseImpl = (new Function('return globalThis;'))().Promise;
 
 /**
  * Get the path to the included file from the parent file path and the
